### DISCUSSION TITLE
Expo and supabase starter template

### DISCUSSION THREAD

**Discussion by Dosbodoke** [electric-sql/electric#1383] [Getting started]
There is any starter template for such setup, using EXPO SDK 51 that standardizes `expo-sqlite`

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
